Doc/library/ctypes.rst

```diff
@@ -487,7 +487,7 @@ single character Python bytes object into a C char::
    >>> strchr(b"abcdef", b"def")
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
-   ArgumentError: argument 2: TypeError: one character string expected
+   ArgumentError: argument 2: TypeError: wrong type
```

Comment:
Also, it should be `ctypes.ArgumentError` here and in a few other places. I can take care of it in a separate PR.